tests/pytests/unit/beacons/test_haproxy.py

```diff
@@ -4,10 +4,14 @@
 
     HAProxy beacon test cases
 """
+import logging
+
 import pytest
 import salt.beacons.haproxy as haproxy
 from tests.support.mock import MagicMock, patch
 
+log = logging.getLogger(__name__)
+
```

Comment:
**question** do we need the import/create log here?

Reply:
It's not being actively used here and can be removed if necessary.